google/ggrc-core
# GGRC-7113 Fix js error on dashboard initialization

# Issue description

App crash for some a user with many permissions (e.g. ~ 100k of Snapshots)

# Steps to test the changes

Prerequisites: user with a lot of permissions.

 - Open Login page 
 - Login to the app

 Actual Result: "Maximum call stack size exceeded" error is displayed. App is freezing.
 Expected Result: No errors should be shown up

# Solution description

 - Do not convert options object into can.Map
 - Do not pass not needed properties to dashboard renderer

# Sanity checklist




# PR Review checklist
